_episodes/04-lists.md

```diff
@@ -160,32 +160,42 @@ does not.
 > ## Nested Lists
 > Since a list can contain any Python variables, it can even contain other lists.
 >
-> For example, we could represent the products in the shelves of a small grocery shop:
+> For example, you could represent the products on the shelves of a small grocery shop
+> as a nested list called x:
+>
+> [![x is represented as a shelf full of produce. There are four rows of vegetables
+> on the shelf, and each row contains five baskets of vegetables. We can label
+> each basket according to the type of vegetable it contains, so the top row
+> contains (from left to right) lettuce, lettuce, peppers, zucchini, squash,
+> and basil.](../fig/04_groceries_x.png)]
```

Comment:
The outer `[` and `]` characters are not needed for the image syntax above or for the next two added images and are currently appearing as literal brackets in the rendered lesson. They were previously being used with the pepper image to create a link to the Hadley Wickham tweet.

Also, the alt text mentions five baskets of vegetables, but currently the images show 6 baskets per row.